src/libais/ais8.cpp

```diff
@@ -626,8 +626,104 @@ Ais8_1_31::Ais8_1_31(const char *nmea_payload, const size_t pad)
   spare2 = bits.ToUnsignedInt(350, 10);
 
   assert(bits.GetRemaining() == 0);
+}
+
+// SSW FI23 Satellite Ship Weather 1-Slot Version
```

Comment:
8_367_* messages should go in ais8_367.cpp

Reply:
FYI: the code for 8:367:33 is quite large, is it okay to have that split out into its own separate file?